### ISSUE TITLE
Allow a manually triggered Jenkins build to form part of the validation

### ISSUE THREAD

**Issue by retronym** [scala/scabot#24]
It would be handy to lend some structure to [comments like](https://github.com/scala/scala/pull/4369#issuecomment-76641232):

> I've [triggered](https://jenkins-dbuild.typesafe.com:8499/job/Community-2.11.x-manual/140/parameters/) a community build based on this PR using `-Ydelambdafy:method`.

Scabot could add a dependency on the status of that build. We would also need a command to remove these dependencies, as we sometimes want to discard a failed run (perhaps wrongly configured) in favour of a new run.

**Comment by adriaanm** [comment]
Yes, that would be relatively straightforward in scabot's current incarnation. All the info it needs to propagate a jenkins result to a commit status is in the job params. It would actually already do this if you set these params manually. See https://github.com/scala/scabot/blob/a5272bf/core/src/main/scala/Core.scala#L32
